activity_simple/my_activities.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'temporalio/activity'
+
+module ActivitySimple
+  module MyActivities
+    # Fake database client
+    class MyDatabaseClient
+      def select_value(table)
```

Comment:
probably worth adding a short sleep here to emphasize that we're faking I/O (as you do below in the message-passing sample)

Reply:
Meh, not really needed that much (the async concepts of whether a sleep is present or not is different in Ruby and doesn't really mean anything) and wasn't done at https://github.com/temporalio/samples-dotnet/blob/main/src/ActivitySimple/MyActivities.cs either where it doesn't matter there either. But I may add this.